tools/recipes/rt-app.recipe

```diff
@@ -9,11 +9,20 @@ build_jsonc() {
     make
 }
 
+build_numactl() {
+    cd numactl
+    ./autogen.sh
+    echo "./configure --host=$CONFIGURE_HOST --disable-shared --enable-static"
```

Comment:
Do we need this echo in the final code ? I'd rather remove it if it's a debug print as it could easily get out of sync with the code. If someone wants to debug, they can easily add `set -o xtrace` or even add an option to do that from `lisa-build-asset` by propagating a `debug` flag turned into `-x` down to:
https://github.com/ARM-software/lisa/blob/master/tools/lisa-build-asset#L73

Reply:
Agree, I will remove it.